float32 VitesseDroite
float32 VitesseGauche
bool    TestSleep

